src/migration-wizard/steps/connect/sources-table/actions/RemoveSourceAction.tsx

```
@@ -1,23 +1,21 @@
   Icon,
-  Text,
   TextContent,
-  Tooltip,
-} from '@patternfly/react-core';
-import { TrashIcon } from '@patternfly/react-icons';
-
-import { ConfirmationModal } from '../../../../../components/ConfirmationModal';
+  Text,
+} from "@patternfly/react-core";
+import { TrashIcon } from "@patternfly/react-icons";
+import { ConfirmationModal } from "../../../../../components/ConfirmationModal";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)